src/App.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
```

Comment:
- [ ] On the mobile view, in the "reservations" section, the styling is questionable:

Kindly, make sure the content does fit and does not overflow the screen. 👍